.github/workflows/merge-tests.yml

```diff
@@ -103,25 +103,29 @@ jobs:
       - name: Run GPU bench on base branch
         run: cd master/benches && just --dotenv-filename bench.env gpu-bench fibonacci_lem
       - name: Copy bench output to PR branch
-        run: cp master/${{ env.BASE_REF }}.json .
+        run: |
+          mkdir -p target
+          cp -r master/target/criterion target
+          cp master/${{ env.BASE_REF }}.json .
       - name: Run GPU bench on PR branch
         run: cd benches && just --dotenv-filename bench.env gpu-bench fibonacci_lem
       # Create a `criterion-table` and write in commit comment
       - name: Run `criterion-table`
         run: cat ${{ github.sha }}.json | criterion-table > BENCHMARKS.md
-      - name: Write bench on commit comment
-        uses: peter-evans/commit-comment@v3
-        with:
-          body-path: BENCHMARKS.md
+      #- name: Write bench on commit comment
+      #  uses: peter-evans/commit-comment@v3
+      #  with:
+      #    body-path: BENCHMARKS.md
       # TODO: Use jq for JSON parsing if needed
       # Check for benchmark regression based on Criterion's configured noise threshold
       - name: Performance regression check
         id: check-regression
         run: |
+          echo $(grep -c 'Regressed' ${{ github.sha }}.json)
```

Comment:
Consider using `echo "foo" | tee -a myfile` instead of  `echo "foo"; echo "foo" >> myfile`.